spec/lib/dotiw_spec.rb

```diff
@@ -3,12 +3,12 @@
 require 'spec_helper'
 
 describe 'A better distance_of_time_in_words' do
+  include DOTIW::Methods
```

Comment:
Once I removed `include DOTIW::Methods` from `lib/dotiw/action_view/helpers/date_helper.rb` a whole slew of specs broke. Once I added this back here those specs started working again.